packages/docs/docs/components/spline.mdx

```diff
@@ -0,0 +1,493 @@
+---
+sidebar_position: 4
+slug: /spline
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Spline
+
+```tsx editor
+import {makeScene2D} from '@motion-canvas/2d/lib/scenes';
+import {Spline, Knot} from '@motion-canvas/2d/lib/components';
+import {createRef} from '@motion-canvas/core/lib/utils';
+import {waitFor} from '@motion-canvas/core/lib/flow';
+
+export default makeScene2D(function* (view) {
+  const spline = createRef<Spline>();
+
+  view.add(
+    <Spline ref={spline} lineWidth={4} fill={'#e13238'} closed>
+      <Knot position={[-120, -30]} startHandle={[0, 70]} />
+      <Knot
+        position={[0, -50]}
+        startHandle={[-40, -60]}
+        endHandle={[40, -60]}
+      />
+      <Knot position={[120, -30]} startHandle={[0, -70]} />
+      <Knot position={[0, 100]} startHandle={[5, 0]} />
+    </Spline>,
+  );
+
+  yield* spline().scale(0.9, 0.6).to(1, 0.4);
+});
+```
+
+The `Spline` component allows us to draw and animate smooth curves through a
```

Comment:
There should be a link to the corresponding API docs at least in the first mention of the node.